docs/tutorials/getting-started-with-node-js.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Getting Started with Node JS
+group: software-languages
+author: "@liam-b"
+---
+
+{% include /style/icon.html type="warning" %}
+Note that Node JS run really slowly on the EV3 brick and is limited to NPM version 0.10.29 which can be a problem when trying to install some libraries.
+{: .alert .alert-warning}
+
+**Before you start, make sure that you have configured a network connection to
+your ev3dev device and have opened an SSH connection to it.**
+
+## Installing ev3dev-lang
+To install the library run:
+
+```shell
+npm install ev3dev-lang
+```
+
+Now whenever you want to use this in code, just write:
+
+```javascript
+var ev3dev = require('ev3dev-lang');
+```
+
+For a bit more info on this visit the ev3dev-lang [github page](https://github.com/wasabifan/ev3dev-lang-js).
+
+## Nano
+For this tutorial, we will be using nano. For more info on how to use nano check out the [nano cheat sheet](/docs/tutorials/nano-cheat-sheet).
+
+## Hello world 
+Navigate to a project directory (e.g `~/src/js`) and make a new file with `touch test.js`
+
+Open it with `nano test.js` and write:
+```javascript
+var test = 'hello world!';
+console.log(test);
+```
+
+Running `node test.js`should print `hello world!` to the console. If this worked you are ready to start coding with node!
+
+## Motors
+Plug a ev3 large motor into port A, then open `test.js` and write:
+
+{% include /style/icon.html type="warning" %}
+This snippet of code will not work for any other motors!
+{: .alert .alert-warning}
+
+```javascript
+var ev3dev = require('ev3dev-lang');
+
+var motor = new ev3dev.LargeMotor('outA'); // create new motor plugged into port A called 'motor'
+
+motor.runForever(200); // run motor at speed 200
+
+setTimeout(function () {
+  motor.stop(); // stop the motor after a second
+}, 1000)
+```
+
+If you get any errors when you try to run this, make sure your motor is plugged in correctly.
+
+For other motors such as the `MeduimMotor`, check the [default supported motors](http://wasabifan.github.io/ev3dev-lang-js/modules/_motors_.html) list.
+
+And from reading the [documentation page](http://wasabifan.github.io/ev3dev-lang-js) you should be able to get a good idea of what methods you can use with the motor (things like `runForever()` and `runForTime()`). Then when you use it in code, it should look like:
+
+```javascript
+motor.doSomething(speed, otherArgument) // where doSomething can be replaced by any valid method.
+```
+
+You can also **reset** and **read** the position of a motor using `motor.reset()` and `motor.position`.
```

Comment:
Keep in mind that `reset` resets the position _and the last-used command, stop mode, etc._ Resetting the position alone is `motor.position = 0`. To make this clear, you could do this:

> You can also **reset** and **read the position of** a motor using `motor.reset()` and `motor.position`, respectively.

What you do exactly depends on what you'd like to communicate there; I'll leave that up to you.